easybuild/easyconfigs/m/MUSCLE/MUSCLE-3.8.31-intel-2018a.eb

```diff
@@ -0,0 +1,36 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'MUSCLE'
+version = '3.8.31'
+
+homepage = 'http://drive5.com/muscle/'
+description = """ MUSCLE is one of the best-performing multiple alignment programs 
+ according to published benchmark tests, with accuracy and speed that are consistently 
+ better than CLUSTALW. MUSCLE can align hundreds of sequences in seconds. Most users 
+ learn everything they need to know about MUSCLE in a few minutes—only a handful of 
```

Comment:
there is a non-ascii dash here (which in my case it's triggering an "ordinal not in range" error when I use easybuild from a git repo but not with an easybuild-installed easybuild)

Reply:
I'll remove the non-ascii dash, but it like to figure out what the exact problem is with having a unicode character in the description...

I can't reproduce the issue with this easyconfig file, even when using current `develop`.

I suspect it has something to do with locale settings; I have both `$LC_ALL` and `$LANG` set to `en_US.utf8`, is that different for you?

Same question to @vanzod who was also hitting this issue...

Reply:
@boegel regarding locale, same here.

For me the problem is not with using `develop` per se, it's when using the module created by the `install-EasyBuild-develop.sh` script, pointing to a git repo. 

If I checkout `eb353` there the problem doesn't go away. Switching python versions between the system's `2.6.6` and a locally compiled `2.7.14` doesn't help either.

But when I use the easybuild-installed `EasyBuild` module, there's no problem...

Reply:
This problem only occurs when EasyBuild is used from a Git repo because of a bug, see https://github.com/easybuilders/easybuild-framework/pull/2472 .